internal/pkg/middleware/middleware.go

```diff
@@ -411,11 +404,13 @@ func (m *Middleware) RequireEvent() func(next http.Handler) http.Handler {
 
 func (m *Middleware) RequireOrganisation() func(next http.Handler) http.Handler {
 	return func(next http.Handler) http.Handler {
-
 		return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-
 			orgID := chi.URLParam(r, "orgID")
 
+			if util.IsStringEmpty(orgID) {
+				orgID = r.URL.Query().Get("orgID")
```

Comment:
Is there any particular reason why we have to do this?

Reply:
There's no `orgID` path variable for the public apis, so it has to be sepcified via query.